### PR TITLE
Support S2 angle for folders (not .SAFE) and Reestructure repository to cookiecutter

### PR DESCRIPTION
**Pull request by marujore**

### CODE REVIEW

File: docs/sphinx/index.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+    Python Client Library for Sentinel-2 Angle Bands is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+
+.. include:: ../../README.rst
+   :end-before: About
+
+
+A client library in Python for generating Sentinel-2 Angle Bands[C.
```
Comment (raphaelrpl) [inline docs/sphinx/index.rst]: There are non printable chars in this line.

File: s2angs/config.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    """Base Configuration."""
+
+    DEBUG = False
+    TESTING = False
+
+    SECRET_KEY = 'secret-key'
+
+    S2ANGS_BASE_PATH_TEMPLATES = os.getenv('S2ANGS_BASE_PATH_TEMPLATES', 'templates')
+
+    S2ANGS_SMTP_PORT = os.getenv('S2ANGS_SMTP_PORT', 587)
```
Comment (raphaelrpl) [inline s2angs/config.py]: What are these variables? Is it valid?

File: s2angs/s2_angs.py [hunk 3]
```diff
@@ -133,165 +143,203 @@ def get_sensor_angles(xml):
...
     """
-    path = os.path.split(xml)[0]
+    path = os.path.split(mtd)[0]
     imgFolder = path + "/IMG_DATA/"
     angFolder = path + "/ANG_DATA/"
     os.makedirs(angFolder, exist_ok=True)
 
     #use band 4 as reference due to 10m spatial resolution
-    imgref = [f for f in glob.glob(imgFolder + "**/*04.jp2", recursive=True)][0]
+    imgref = [f for f in glob.glob(imgFolder + "**/*04*.jp2", recursive=True)]
```
Comment (raphaelrpl) [inline s2angs/s2_angs.py:196]: Since you are seeking for `.jp2` files, you should consider to add a validation here since it may be empty.

**Review by raphaelrpl — CHANGES_REQUESTED** [review]

File: s2angs/config.py (hunk 2 above)
Comment (marujore) [reply s2angs/config.py]: The file has been removed

File: s2angs/s2_angs.py (hunk 3 above)
Comment (marujore) [reply s2angs/s2_angs.py:196]: Agree. Solved.

**Pull request merged**